tests/dtm/test_rgw_s3_process_restart.py

```diff
@@ -496,3 +497,152 @@ def test_continuous_write_during_rgw_s3_restart(self):
         self.test_completed = True
 
         self.log.info("ENDED: Verify continuous WRITEs during rgw_s3 restart using pkill")
+
+    # pylint: disable-msg=too-many-locals
+    # pylint: disable-msg=cell-var-from-loop
+    @pytest.mark.lc
+    @pytest.mark.dtm
+    @pytest.mark.tags("TEST-42249")
+    def test_cont_delete_during_rgw_svc_restart(self):
+        """Verify continuous DELETEs during rgw_s3 service restart using pkill."""
+        self.log.info("STARTED: Verify continuous DELETEs during rgw_s3 service restart using "
+                      "pkill")
+        test_prefix = 'test-42249'
+        wr_output = Queue()
+        test_cfg = DTM_CFG["test_42246"]
```

Comment:
Is it correct? do you want to take the config from test_42246?

Reply:
Yes, this is correct.